AirCasting/SDSync/ViewsFlow/SDRestartAB/SDRestartABViewModel.swift

```diff
@@ -6,6 +6,9 @@ import Foundation
 protocol SDRestartABViewModel: ObservableObject {
     var presentNextScreen: Bool { get set }
     var urlProvider: BaseURLProvider { get }
+    // urlProvider should should not be exposed
+    // BUT it is - REASON: it is needed only to pass to some navigation view
```

Comment:
the usual convention is to put explanation comments above the line being explained, not below